### PR TITLE
New coredns version 1.6.2

### PR DESCRIPTION
**Pull request by pipo02mix**
Include new coreDNS version 1.6.2

### CODE REVIEW

**Comment by jgsqware** [comment]
there is an PM (I will create it) wher we need to change the default port to 1053.
Can this be included in this one? 

@corest

**Comment by jgsqware** [comment]
Thanks @corest

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,22 @@ All notable changes to this project will be documented in this file.
...
+## [v0.7.0]
+
+### Added
+
+- Change CoreDNS version to `1.6.2` with different enhancements and fixes.
+  - [1.6.0 release notes](https://coredns.io/2019/07/28/coredns-1.6.0-release/).
+  - [1.6.1 release notes](https://coredns.io/2019/08/02/coredns-1.6.1-release/).
+  - [1.6.2 release notes](https://coredns.io/2019/08/13/coredns-1.6.2-release/).
+
+- The deployment has included the Prometheus Operator annotations to make easier the target discovery by Prometheus.
```
Comment (MarcelMue) [inline CHANGELOG.md]:
```suggestion
- The deployment has included the Prometheus Operator annotations to make the target discovery easier by Prometheus.
```

**Review by MarcelMue — APPROVED** [review]
I was looking at upgrading coredns right now as well haha.

**Pull request merged**